nginx/templates/nginx.conf.template

```diff
@@ -25,7 +25,7 @@ http {
       # server unix:/run/gunicorn.sock fail_timeout=0;
 
       # for a TCP configuration
-      server web:8000 fail_timeout=0;
+      server web:${GUNICORN_BIND_PORT} fail_timeout=0;
```

Comment:
The location and name of the file is important for the nginx image.

Location – used as a volume in docker-compose
Name – nginx will look for `.template` files and generate a file with the suffix removed. In this case `nginx.conf`